@xen-orchestra/fs/package.json

```diff
@@ -21,7 +21,7 @@
     "node": ">=6"
   },
   "dependencies": {
-    "@marsaud/smb2": "^0.13.0",
+    "@marsaud/smb2": "^0.14.0",
```

Comment:
Either use `yarn update-interactive`, or run `yarn` if you update deps manually, otherwise `yarn.lock` is not up-to-date.

Reply:
Don't worry about this for this PR, I'll take care of it.